src/Symfony/Component/Security/Http/RememberMe/AbstractRememberMeServices.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\Security\Http\RememberMe;
 
+use Exception;
```

Comment:
we don't "use" root classes on the code base, so you should use `\Exception` everywhere instead.